packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -217,8 +217,10 @@ abstract class NavigationGestureController {
   void dragUpdate(double fractionalDelta);
 
   /// The drag gesture has ended with a horizontal motion of
-  /// [fractionalVelocity] as a fraction of screen width per second.
-  void dragEnd(double fractionalVelocity);
+  /// [fractionalVelocity] as a fraction of screen width per second. Returns
```

Comment:
I'd put the sentence that starts with "Returns" on a new line after a blank (comment) line.  That will make the docs look slightly prettier on the web site:

```dart
/// [fractionalVelocity] as a fraction of screen width per second.
///
/// Returns...
```